content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md

```diff
@@ -17,13 +17,36 @@ weight: 1
 
 # Corda: Enterprise Edition release notes
 
+## Corda: Enterprise Edition 4.9.2
+
+Corda Enterprise 4.9.2 is a patch release of Corda Enterprise which includes the addition of a new Gradle 7 plugin, security upgrades, and fixes for minor bugs.
+
+* A new version of the Gradle plug-in that works with [Gradle 7](https://docs.gradle.org/7.0/release-notes.html) has been released as part of this patch. The plugin can be found in [Jenkins](https://ci01.dev.r3.com/blue/organizations/jenkins/Corda-Gradle-Build-Plugins%2Fcorda-gradle-plugins/detail/release%2F5.1.0-RC01/1/pipeline/).
+For more information on how to use the plugin, visit its [GitHub repository](https://github.com/corda/samples-kotlin/tree/chrisr3-gradle7).
+
+### Upgrade recommendation
+
+As a developer, you should upgrade to the [latest released version of Corda](../../../../../en/platform/corda/4.9/enterprise.html) as soon as possible. The latest Corda Enterprise release notes are on this page, and you can find the latest upgrade guide [here](../../../../../en/platform/corda/4.9/enterprise/upgrading-index.md).
+
+As a node operator, you should upgrade to the [latest released version of Corda](../../../../../en/platform/corda/4.9/enterprise.html).
+
+### Fixed issues
+
+In this patch release:
+
+* The `corda-shell --version` command now always returns the correct version.
+* The time it takes for first-time flows to be run in a signed CorDapp has been significantly reduced.
```

Comment:
This fixes a regression introduced in Corda version...? (By [CORDA-4120](https://r3-cev.atlassian.net/browse/CORDA-4120))